encord/dataset.py

```diff
@@ -84,8 +92,9 @@ def create_image_group(self, file_paths: Iterable[str], max_workers: Optional[in
             file_paths: a list of paths to images, e.g.
                 ['/home/user/data/img1.png', '/home/user/data/img2.png']
             max_workers:
-                Number of workers for parallel image upload. If set to None, this will be the number of CPU cores
-                available on the machine.
+                DEPRECATED: This argument will be ignored
```

Comment:
A more gentle deprecation strategy would be to read the max_workers and create a CloudUploadSettings with it, so that we don't alter existing behaviour

Reply:
There are no max_workers anymore. I am not sure I understand